packages/transmute-cli/src/commands/generate-recovery-key/index.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+
+const openpgp = require('openpgp');
+
+const run = require('../runner');
+
+const shell = require('shelljs');
+
+const { writeFile, readFile, removeFile } = require('../../utils');
+
+module.exports.generateRecoveryKey = async (args) => {
+
+  let transmuteDir = path.join(require('os').homedir(), '.transmute/cli-secrets');
+  shell.env.RECOVERY_KEY_FINGERPRINT = args.fingerprint;
```

Comment:
cool, did not know we can do this.

Reply:
Still wonder if this is better to be a script argument instead of env var.

Reply:
Updated.